enterprise/frontend/src/embedding-sdk/test/sdk-compatibility-warning.unit.spec.tsx

```diff
@@ -23,7 +23,7 @@ import { getEmbeddingSdkVersion } from "../config";
 const METABASE_INSTANCE_URL = "path:";
 const AUTH_PROVIDER_URL = "http://auth-provider/metabase-sso";
 
-const defaultAuthUriConfig = defineEmbeddingSdkConfig({
+const defaultAuthUriConfig = defineEmbeddingSdkAuthConfig({
```

Comment:
I'd suggest we rename this to:

```suggestion
const defaultAuthConfig = defineEmbeddingSdkAuthConfig({
```

or

```suggestion
const defaultSdkAuthConfig = defineEmbeddingSdkAuthConfig({
```

based on `DEFAULT_SDK_AUTH_PROVIDER_CONFIG` and `DEFAULT_AUTH_CONFIG` that we use in this PR.